src/main/java/io/redisearch/Document.java

```diff
@@ -14,8 +14,10 @@
  */
 public class Document implements Serializable {
 
+    private static final long serialVersionUID = 2L;
```

Comment:
@sazzad16 it's better to set the serialVersionUID value that was auto generated by the previous version